manifests/init.pp

```diff
@@ -0,0 +1,72 @@
+# Installation of initial dependencies and baseplate project
+# in a development environment.
+# Please add additional dependencies and env setup code as needed.
+class baseplate {
+  $project_path = '/home/vagrant/baseplate'
+
+  group { 'puppet':
```

Comment:
this was copied from the manifest I linked as an example in the cookiecutter template https://github.com/reddit/reddit-cookiecutter-pyramid/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/manifests/init.pp

I think that was erroneously copied from an example I used as I was learning how to write puppet manifests properly. it shouldn't be needed here (or there).